index.html

```diff
@@ -6960,22 +6960,22 @@ <h1>Join us at the FOSSASIA OpenTechSummit
                         <a target="_self" class="btn btn-lg" href="http://2018.fossasia.org/tickets">Tickets</a>
                         <span class="uppercase">Oh, invite your friends too</span>
                         <a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=fossasia.org">
-                            <i class="icon social_facebook icon-large"></i>
+                            <i class="icon social_facebook icon-large responsive_icons"></i>
```

Comment:
Instead of adding a margin to all icons, add the margin to the container.

Reply:
@abishekvashok No, that won't work. The issue is that when icons moves to the next row, then the margin is absent. Adding margin to the container still won't solve the problem here.

Reply:
@akshatnitd exactly. Initially I tried out what @abishekvashok said but the issue remained. Then I resolved this issue in the present way.